docs/Developers/Writing_Native_Bears.rst

```diff
@@ -51,14 +51,15 @@ each file:
 
 .. code:: python
 
-    from coalib.bears.LocalBear import LocalBear
+    import logging
 
+    from coalib.bears.LocalBear import LocalBear
```

Comment:
I've finally modified the imports' order based on a suggestion from `isort`: I think it would improve a little bit clarity :)